app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+const mongoose = require('mongoose');
+const url = 'mongodb+srv://Immanuel50:chukaglorY55@cluster0-ggn2t.mongodb.net/bucket-list-app?retryWrites=true&w=majority';
```

Comment:
Please move this secret key to your `env` file. You should never expose confidential information. 

eg
in your `.env` file create a constant called `MONGOOSE_URL=` then import that here